Sources/BezierSwift/MasterComponent/BezierDialog/BezierDialog.swift

```diff
@@ -6,15 +6,229 @@
+      
+      if let buttonInfo = param.buttonInfo {
+        switch buttonInfo {
+        case .vertical(let buttons):
+          self.buttons = buttons
+          self.isButtonStackVertical = true
+        case .horizontal(let buttons):
+          self.buttons = buttons
+          self.isButtonStackVertical = false
+        }
+      }
+    }
+    
+    self.isPresented = true
+  }
+  
+  func dismiss() {
+    self.currentDismissCompletion?()
+    
+    self.currentDismissCompletion = nil
+    self.currentParamId = UUID()
+    self.currentPriority = Int.min
+    
+    self.isPresented = false
+  }
+}
+
+class BezierDialogSingleton {
+  var viewModel: DialogViewModel
+  
+  static let shared = BezierDialogSingleton()
+
+  init() {
+    self.viewModel = DialogViewModel()
+  }
 }
+
+@available(iOS 15.0, *)
+struct BezierDialog: ViewModifier, Themeable {
+  @Environment(\.colorScheme) public var colorScheme
+  
+  @StateObject private var viewModel: DialogViewModel = BezierDialogSingleton.shared.viewModel
+  
+  
+  func body(content: Content) -> some View {
+    
+    content
+      .overlay {
+        if viewModel.isPresented {
+          self.palette(.bgtxtAbsoluteBlackLighter)
+            .edgesIgnoringSafeArea(.all)
+            .overlay {
+              VStack(alignment: .center, spacing: .zero) {
+                Spacer()
+                HStack(spacing: .zero) {
+                  Spacer()
+
+                  HStack(spacing: .zero) {
+                    Spacer()
+                    VStack(alignment: .center, spacing: Metric.belowStackTop) {
+                      if !viewModel.title.isEmpty || !viewModel.description.isEmpty {
+                        VStack(alignment: .center, spacing: Metric.upperStackSpace) {
+                          Text(viewModel.title)
+                            .applyBezierFontStyle(.bold16)
+                          Text(viewModel.description)
+                            .applyBezierFontStyle(.normal14)
+                        }
+                        .padding(.top, Metric.upperStackContainerTop)
+                      }
+
+                      if !viewModel.buttons.isEmpty {
+                        if viewModel.isButtonStackVertical {
+                          VStack(spacing: Metric.belowStackSpace) {
+                            ForEach(viewModel.buttons.prefix(4).indices, id: \.self) { idx in
+                              viewModel.buttons[idx]
+                            }
+                          }
+                        } else {
+                          HStack(spacing: Metric.belowStackSpace) {
+                            ForEach(viewModel.buttons.prefix(2).indices, id: \.self) { idx in
+                              viewModel.buttons[idx]
+                            }
+                          }
+                        }
+                      }
+                      
+                    }
+                    .padding(.all, Metric.dialogPadding)
```

Comment:
```suggestion
                    .padding(.all, Metric.dialogPadding)
                    .frame(maxWidth: .infinity)
```
위에 코드랑 같이 추가해주면 Spacer() 전부 제거가 가능할 것 같습니다.